cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -0,0 +1,189 @@
+private import cpp
+private import DataFlowUtil
+
+/**
+ * A data flow node that occurs as the argument of a call and is passed as-is
+ * to the callable. Arguments that are wrapped in an implicit varargs array
+ * creation are not included, but the implicitly created array is.
+ * Instance arguments are also included.
+ */
```

Comment:
Parts of this comment are Java-specific. I can see the same copy-paste error exist in the file you copied from.